test/cppsuite/src/component/timestamp_manager.cpp

```diff
@@ -45,6 +46,21 @@ timestamp_manager::decimal_to_hex(uint64_t value)
     return (res);
 }
 
+uint64_t
+timestamp_manager::hex_to_decimal(const char *s)
```

Comment:
Instead of a `const char *` can we use `std::string` to be consistent with the above function? Feels more characteristic of cpp